caffe2/onnx/onnxifi_manager.cc

```diff
@@ -0,0 +1,55 @@
+#include "caffe2/onnx/onnxifi_manager.h"
+
+#include <mutex>
+
+#include "caffe2/core/logging.h"
+
+namespace caffe2 {
+namespace onnx {
+
+onnxifi_library* OnnxifiManager::AddOnnxifiLibrary(
+    const std::string& name) {
+  auto it = onnxifi_interfaces_.find(name);
```

Comment:
I think you need a lock for the `onnxifi_interfaces_` since the manager is a singleton